sdk/hardwaresecuritymodules/arm-hardwaresecuritymodules/CHANGELOG.md

```diff
@@ -1,5 +1,131 @@
+  - Added Interface PrivateLinkServiceConnectionState
+  - Added Interface ProxyResource
+  - Added Interface RestoreRequestProperties
+  - Added Interface RestoreResult
+  - Added Interface TrackedResource
+  - Added Interface UserAssignedIdentity
+  - Interface ApiEntityReference has a new optional parameter resourceId
+  - Interface NetworkInterface has a new optional parameter resourceId
+  - Interface Resource has a new optional parameter systemData
+  - Added Type Alias ActionType
+  - Added Type Alias ActivationState
+  - Added Type Alias AutoGeneratedDomainNameLabelScope
+  - Added Type Alias BackupRestoreOperationStatus
+  - Added Type Alias CloudHsmClusterSkuFamily
+  - Added Type Alias CloudHsmClusterSkuName
+  - Added Type Alias CreatedByType
+  - Added Type Alias ManagedServiceIdentityType
+  - Added Type Alias Origin
+  - Added Type Alias PrivateEndpointConnectionProvisioningState
+  - Added Type Alias PrivateEndpointServiceConnectionStatus
+  - Added Type Alias ProvisioningState
+  - Added Type Alias PublicNetworkAccess
+  - Added Enum KnownActionType
+  - Added Enum KnownActivationState
+  - Added Enum KnownAutoGeneratedDomainNameLabelScope
+  - Added Enum KnownBackupRestoreOperationStatus
+  - Added Enum KnownCloudHsmClusterSkuFamily
+  - Added Enum KnownCreatedByType
+  - Added Enum KnownManagedServiceIdentityType
+  - Added Enum KnownOrigin
+  - Added Enum KnownPrivateEndpointConnectionProvisioningState
+  - Added Enum KnownPrivateEndpointServiceConnectionStatus
+  - Added Enum KnownProvisioningState
+  - Added Enum KnownPublicNetworkAccess
+  - Added Enum KnownVersions
+### Breaking Changes
+  - Class AzureDedicatedHSMResourceProvider no longer has parameter $host
+  - Class AzureDedicatedHSMResourceProvider no longer has parameter apiVersion
+  - Class AzureDedicatedHSMResourceProvider no longer has parameter dedicatedHsmOperations
+  - Class AzureDedicatedHSMResourceProvider no longer has parameter subscriptionId
+  - Removed Interface DedicatedHsmListResult
+  - Removed Interface DedicatedHsmOperation
+  - Removed Interface DedicatedHsmOperationDisplay
+  - Removed Interface DedicatedHsmOperationListResult
+  - Removed Interface DedicatedHsmOperationsOperations
+  - Removed Interface OutboundEnvironmentEndpointCollection
+  - Removed Interface ResourceListResult
+  - Interface ApiEntityReference no longer has parameter id
+  - Interface AzureDedicatedHSMResourceProviderOptionalParams no longer has parameter $host
+  - Interface AzureDedicatedHSMResourceProviderOptionalParams no longer has parameter endpoint
+  - Interface DedicatedHsmCreateOrUpdateOptionalParams no longer has parameter resumeFrom
+  - Interface DedicatedHsmDeleteOptionalParams no longer has parameter resumeFrom
+  - Interface DedicatedHsmUpdateOptionalParams no longer has parameter resumeFrom
+  - Interface DedicatedHsmUpdateOptionalParams no longer has parameter tags
+  - Interface NetworkInterface no longer has parameter id
```

Comment:
why this?

Reply:
These are the api version breakings should be introduced by https://github.com/Azure/azure-rest-api-specs/pull/26326